docs/BackupAndRestore.md

```diff
@@ -0,0 +1,594 @@
+
+- explicitly selecting specific archive IDs - `restore.archiveIds` variable can be set to specific archive IDs in the domain state for the target database.
+To list all archives for a database, the `nuocmd show archives --db-name <db>` is used.
+The process for each archive can be seen under the archive info which makes it easier for the user to mark the archives for restore.
+- selecting archives served by database processes with specific labels - `restore.labels` can be used to configure process labels, which then define the archives which will be selected for a restore.
+Any configured label and value that matches an SM process will add its archive to the list of selected archives for a restore.
+For instance, this can be used to easily select all SMs in a specific backup group - `--set restore.labels.backup="<backup group>"`.
+
+### Archive seed restore
+
+The _archive seed restore_ operation restores a corrupted or lost archive while the database is running.
+This operation will reset the existing archive state, and once it joins the database, the new SM will sync to the current state of the running database.
+For this reason, you cannot restore the entire database by restoring a single SM in a running database.
+The _seed restore_ has the potential to reduce the _SYNCing_ time from other running SMs as only the changed atoms will be transferred.
+The database will remain running throughout this operation.
+
+The high-level steps to perform _archive seed restore_ are the following:
+
+1. Identify which archives will be restored.
+2. Ensure that the restore source is available either locally in the backup volume or as a remote URL.
+3. Ensure that there is enough free disk space in the archive volume of each SM so that a copy of the archive before the restore is taken.
+4. Invoke the database restore request by installing the NuoDB _restore_ chart by specifying the archive IDs for the selected archives.
+5. Start or restart the selected SM
+
+In this example, there is an SM exited because it experienced archive data corruption.
+One way to fix the issue is to completely delete the corrupted archive and let the SM sync the whole archive from one of the running processes in the database.
+To reduce the sync time, we can upload one of the recent online database backups to a remote location and restore the corrupted archive.
+
+> **NOTE**: A copy of an archive or backup set from another database can't be used to perform _archive seed restore_. Otherwise the Storage Manager process will fail to start with error _Archive "/var/opt/nuodb/archive/nuodb/demo" doesn't match database.  Expected UUID \*\*\*, got \*\*\*._
+
+Identify its archive id using `nuocmd show archives` commands and select it for restore by installing the _restore_ chart with `restore.type="archive"`.
```

Comment:
Do labels work for seed restore? `nuodocker start sm` injects the label `pod-name`, so specifying `--set restore.labels.pod-name <pod name>` might be a more convenient way to issue the seed restore command (assuming that it works).

Reply:
Good idea! That should work. I will test and use that instead.